src/Symfony/Component/Validator/Constraints/Ulid.php

```diff
@@ -35,20 +35,27 @@ class Ulid extends Constraint
         self::TOO_LARGE_ERROR => 'TOO_LARGE_ERROR',
     ];
 
+    public const VALIDATE_BASE_32 = 'base32';
+    public const VALIDATE_BASE_58 = 'base58';
+
     public string $message = 'This is not a valid ULID.';
+    public string $format = self::VALIDATE_BASE_32;
 
     /**
      * @param array<string,mixed>|null $options
      * @param string[]|null            $groups
+     * @param self::VALIDATE_*|null    $format
```

Comment:
shouldn't the constants be named `FORMAT_*` instead as they represent the available formats ?

Reply:
Right! It's updated